src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -103,6 +115,20 @@ const Authenticate = (() => {
             $('#expiry_datepicker_proofid').setVisibility(0);
             $('#exp_date_2').datepicker('setDate', '2099-12-31');
         }
+
+        if (is_nigeria) {
+            const $nigeria_file_selector = document.querySelector('[data-type="nigeria"]');
+            document.querySelector('[data-type="poi"]').setVisibility(0);
+            $show_onfido_btns = $nigeria_file_selector.querySelectorAll('h3[data-show-onfido]');
+            $show_onfido_btns.forEach($el => {
+                const data_show_onfido = $el.getAttribute('data-show-onfido');
+                if (data_show_onfido) {
+                    $el.addEventListener('click', showOnfido);
```

Comment:
why do we want to intialize onfido at this point?, afaik, nigeria client will fallback to unsupported poi right?